apps/versions/tests.py

```diff
@@ -909,8 +924,14 @@ def test_owner_should_be_allowed(self):
         eq_(response.status_code, 200)
         assert response[settings.XSENDFILE_HEADER]
         assert 'Content-Disposition' in response
-        assert self.filename in response['Content-Disposition']
-        eq_(response[settings.XSENDFILE_HEADER], self.version.source.path)
+        filename = self.filename
+        if not isinstance(filename, unicode):
```

Comment:
Not happy with all those, that I needed to add for the tests to run correctly on travis (they were running fine locally, but raised unicode errors on travis). Not sure why though.